source/sink-connector/configuration-properties/write-strategies.txt

```diff
@@ -23,6 +23,15 @@ configuration:
 
    writemodel.strategy=<a writemodel strategy>
 
+You can also select a delete write model strategy to specify how the
+sink connector writes data into MongoDB when it receives a
+tombstone event. You can specify a delete write strategy with the following
```

Comment:
Issue:
The [style guide](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-using) mentions:
> Use by using, or recast the sentence. Don't use using (alone) or with. By using is easier for worldwide audiences to understand.

Suggestion:
"...strategy by using the..."